analysis/druidrestoration/src/Guide.tsx

```diff
@@ -139,7 +148,7 @@ function EfflorescenceSubsection({ modules, events, info }: GuideProps<typeof Co
       is extremely mana efficient if you're good about placing it where raiders are standing. Under
       the boss is usually a safe bet. While it's acceptable to let it drop during heavy movement,
       you should otherwise aim to keep it active at all times.
-      <br />
+      <p />
```

Comment:
Mostly because I'm a heathen who doesn't know how to do inline styling well >_<

I starting using them because I liked the look of the "line-and-a-half" break it gave me - bigger than one `br` but smaller than two `br`. It sounds like I could get the same effect by wrapping in a paragraph tag, or maybe the margins set thing you suggested?